src/app/cat-filter/cat-filter.component.html

```diff
@@ -0,0 +1,27 @@
+<form [formGroup]="catSelects">
+  <mat-form-field>
+    <mat-select formControlName="category">
+      <mat-option [value]="''"></mat-option>
+      <mat-option
+        *ngFor="let category of (categoriesList$ | async); let i = index"
+        [value]="category.id"
+      >{{ category.name | titlecase }}</mat-option>
+    </mat-select>
+  </mat-form-field>
+  <mat-form-field>
+    <mat-select formControlName="breeds">
+      <mat-option [value]="''"></mat-option>
+      <mat-option
+        *ngFor="let breed of (breedsList$ | async)"
+        [value]="breed.id"
+      >{{ breed.name | titlecase }}</mat-option>
+    </mat-select>
+  </mat-form-field>
+  <mat-paginator
+    #paginator
+    aria-label="Select page"
+    [length]="contentsQuantity$ | async"
+    [pageSize]="pagination.limit"
+    [pageSizeOptions]="[5, 10, 20, 30]"
+  ></mat-paginator>
```

Comment:
I didn't work a lot with paginator, but is there any way to implement it also through formControl